### DISCUSSION TITLE
SNAP Specific Assistance

### DISCUSSION THREAD

**Discussion by DBashirTEALS** [TEALSK12/introduction-to-computer-science#305] [General]
Hi All,

Thanks for visiting the TEALS Intro to CS Github.  My name is Dr. Bashir and I am here to provide a hand in discussions for all things SNAP!

Looking forward to all the great conversation to come!

Cheers,
Dr. Bashir

**Comment by RandyCHS** [comment]
Hi Dr. Bashir,

Thanks for creating this new forum.  I assume that this GitHub Discussions will be replacing the previous forums on Discourse?  
Also I want to invite all teachers and volunteers who are new or experienced with [Snap!](https://snap.berkeley.edu/) to attend [Snap!Con 2021](https://www.snapcon.org/conferences/2021).  I attended last year and learned lots a cool things about Snap!.  It starts in 2 days!

Randy Macdonald

**Reply by DBashirTEALS** [reply]
Hi Randy,

  Thanks for commenting!  Yes the Github Discussions will be replacing Discourse Forums.  As it stands, this board will be focused specifically on SNAP remarks geared to classroom implementation (how to teach something, ideas for projects, asking questions etc.).  Thing like job posting and conferences, I try to keep clear of this space so that teachers and volunteers looking for SNAP specific assistance can receive it.  Thank you for sharing this opportunity with everyone and I'll see to it that it gets shared in other places.  Thanks!

Dr. Bashir

**Reply by RandyCHS** [reply]
Thanks. :-)

**Comment by jdonwells** [comment]
So...The doctor is in?
